src/content/docs/en/guides/integrations-guide/netlify.mdx

```diff
@@ -244,6 +244,39 @@ With [fine-grained cache control](https://www.netlify.com/blog/swr-and-fine-grai
 standard caching headers like `CDN-Cache-Control` or `Vary`.
 Refer to the docs to learn about implementing e.g. time to live (TTL) or stale while revalidate (SWR) caching: https://docs.netlify.com/platform/caching
 
+### Including files on SSR
+
+When using Server-Side Rendering (SSR), you may need to include specific files in your Netlify Functions that aren't automatically bundled during deployment.
+The `includeFiles` property allows you to explicitly specify which files should be bundled with your function.
```

Comment:
```suggestion
The `includeFiles` property allows you to explicitly specify additional files that should be bundled with your function.
```
You don't specify *all the* files, just *additional* ones, right? I think this is more accurate!